### PR TITLE
fix: executeScript return no log

### PR DESCRIPTION
**Pull request by medjedovicm**
## Intent

At one point of `return` inside of `executeScript` function, log was not being returned.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [x] All unit tests are passing (`npm test`).
image
- [x] All `sasjs-cli` unit tests are passing (`npm test`).
NOTICE: it is passing, but there is breaking change in return value of a function `startComputeJob()`. Without that fix in the cli, JOB tests were failing.
image
- [x] All `sasjs-tests` are passing (instructions available [here](https://github.com/sasjs/adapter/blob/master/sasjs-tests/README.md)).
sasjs-tests

### CODE REVIEW

File: src/SASViyaApiClient.ts [hunk 1]
```diff
@@ -589,7 +589,7 @@ export class SASViyaApiClient {
       if (expectWebout) {
         resultLink = `/compute/sessions/${executionSessionId}/filerefs/_webout/content`
       } else {
-        return currentJob
+        return { result: currentJob, log }
```
Comment (krishna-acondy) [inline src/SASViyaApiClient.ts:592]: We're returning the `currentJob` object as `job` on line 584:
https://github.com/sasjs/adapter/pull/169/files#diff-72e1fde4c9683400219b3188eac4c2446ec1cfb314e073ef4f5640e0784c207fR584
So we should probably do the same here.

**Review by krishna-acondy — CHANGES_REQUESTED** [review]

File: src/SASViyaApiClient.ts (hunk 1 above)
Comment (medjedovicm) [reply src/SASViyaApiClient.ts:592]: Good point

File: sasjs-tests/src/testSuites/Compute.ts [hunk 2]
```diff
@@ -25,17 +25,72 @@ export const computeTests = (adapter: SASjs): TestSuite => ({
...
+        const expectedLogContent = `1    %abort;\\nERROR: The %ABORT statement is not valid in open code.\\n`
+        
+        return validateLog(expectedLogContent, res.log);
       }
     }
   ]
 });
 
+const validateLog = (text: string, log: string): boolean => {
+  console.log(`[JSON.stringify(log)]`, JSON.stringify(log))
```
Comment (YuryShkoda) [inline sasjs-tests/src/testSuites/Compute.ts:83]: remove console.log

File: src/SASjs.ts [hunk 3]
```diff
@@ -206,11 +206,20 @@ export default class SASjs {
     return await this.sasViyaApiClient!.createSession(contextName, accessToken)
   }
 
+  /**
+   * Executes the sas code against given sas server
+   * @param fileName - name of the file to run.
+   * @param linesOfCode - lines of sas code from the file to run.
+   * @param contextName - context name override on which code will be run.
+   * @param accessToken - the access tokne to authorizing the request.
+   * @param debug - debug flag override
```
Comment (YuryShkoda) [inline src/SASjs.ts:215]: pls provide more description

**Review by YuryShkoda — CHANGES_REQUESTED** [review]

**Review by allanbowe — APPROVED** [review]

**Review by saadjutt01 — APPROVED** [review]

**Pull request merged**